Content/Items/Consumables/Maps/BossMaps/QueenSlimeMap.cs

```diff
@@ -0,0 +1,35 @@
+﻿using PathOfTerraria.Common.Subworlds.BossDomains.Prehardmode;
+using PathOfTerraria.Core.Items;
+using SubworldLibrary;
+using Terraria.Localization;
+
+namespace PathOfTerraria.Content.Items.Consumables.Maps.BossMaps;
+
+internal class QueenSlimeMap : Map
+{
+	public override int MaxUses => GetBossUseCount();
+	public override int WorldTier => 50;
```

Comment:
This effects area level right?
I'd say have the area level of the hardmode bosses be the tier at which they could be farmed for completion + 1

So in this case a tier 1 map is farmed to get completion, so 52+1